valhalla/app/javascript/components/Thumbnails.vue

```diff
@@ -146,7 +152,13 @@ export default {
     uploadFile: function (event) {
       document.getElementById('browse_everything').click();
     }
-  }
+  },
+  // created () {
```

Comment:
Remove these?

Reply:
This is done in the latest commit. Also rebased, should be ready for merge.